common/util/file.py

```diff
@@ -24,6 +24,7 @@ def determine_syntax_files():
 def get_syntax_for_file(filename):
     extension = get_file_extension(filename)
     syntaxes = syntax_file_map.get(filename, None) or syntax_file_map.get(extension, None)
+    # watch out if this one has changed https://github.com/sublimehq/Packages/tree/master/Text
```

Comment:
It was meat like a comment to the developers that may search for '.tmLanguage' and replacing those with `.sublime-syntax`.

It will not change to sublime-syntax in near future to keep backward compatibility.